archlinux/PKGBUILD-qubes-vm-kernel-support.install

```diff
@@ -9,19 +9,36 @@ help() {
 
 ## arg 1:  the new package version
 post_install() {
+  local begin='### BEGIN QUBES HOOKS ###' end='### END QUBES HOOKS ###'
+  local combined="\\
+$begin\\
+# Set the kernel command line to what the QubesOS initramfs hook expects.\\
+# If the kernel command line is wrong, the system will not boot.\\
+\\
+. /etc/default/grub.qubes\\
```

Comment:
Actually, this may be a wrong place - this file is provided by qubes-core-agent (which, as @icequbes1 noticed, doesn't install it on Arch). On the other hand, this package provides `/etc/default/grub.qubes-kernel-vm-support`, which is not included here.
It may be ok to include both in this install script, but first `qubes-core-agent` needs to install the actual file.